ansible-dk-cli/ansible-dk

```diff
@@ -6,13 +6,44 @@ import click
 import os
 import sys
 import json
+import yaml
+from tempfile import NamedTemporaryFile
+from ansible.executor import playbook_executor
+from ansible.inventory import Inventory
+from ansible.parsing.dataloader import DataLoader
+from ansible.vars import VariableManager
+from collections import namedtuple
 
 ###################################################
 # Defaults
 RUBY_ABI = "2.1.0"
 PYTHON_ABI = "2.7"
 HOME = os.getenv("HOME")
 
+def get_config(debug=False):
+    #TODO: pass debug properly
+    config_path=os.path.expanduser('~/.ansible-dk/config')
+    #TODO: absolute path for default generator repo
+    config = {
+        'generator': {
+            'repo': '/opt/ansible-dk/generators/'
```

Comment:
We're calling it a 'generator', not a generator repo.  Why not move the setting up a level?

generator: '/opt/ansible-dk/generators/default'